src/pages/Login/User.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+
+const User = ({ content, isSelectLogin }) => {
+  const { title, btnText, img } = content;
+  const [info, setInfo] = useState({
+    userId: '',
+    userPw: '',
+  });
+
+  const onChangeinfo = e => {
+    const { name, value } = e.target;
+    setInfo({ ...info, [name]: value });
+  };
+  const navigate = useNavigate();
+
+  const passed =
+    info.userId.length > 5 &&
+    17 > info.userPw.length > 5 &&
+    info.userId.includes('@', '.');
+
+  const toSignUp = e => {
+    e.preventDefault();
+    fetch('http://10.58.2.51:3000/users/signup', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        email: info.userId,
+        password: info.userPw.toString(),
+      }),
+    }).then(response => {
+      if (response.status === 200) {
+        localStorage.setItem('token', String(response.accessToken));
+        alert('회원가입 성공');
+        navigate('/login');
+      }
+      return response.json();
+    });
+  };
+
+  const toLogin = e => {
+    e.preventDefault();
+    fetch('http://10.58.2.51:3000/users/signin', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        email: info.userId,
+        password: info.userPw.toString(),
+      }),
+    }).then(response => {
+      if (response.status === 200) {
+        localStorage.setItem('token', String(response.accessToken));
+        alert('로그인 성공');
+        navigate('/main');
+      } else if (response.status === 400) {
+        alert('아이디 혹은 비밀번호를 확인 해 주세요');
+      }
+      return response.json();
+    });
+  };
+
+  return (
+    <div className="layout">
```

Comment:
- 전체적으로 최상위 태그의 className을 컴포넌트 이름과 동일하게 부여한 뒤 그 className하위로 네스팅 하는 부분이 적용안되어있습니다.
- CRA로는 index.html이 하나인 SPA프로젝트를 구성하는 것이고 이런 상황에서 결과적으로 모든 css파일들은 하나의 html에 한번에 불러와집니다.
- 결과적으로 모든 css파일이 한번에 불러와지기에 이 css파일에서 부여한 스타일이 동일한 className을 가진 다른 컴포넌트의 요소들에게도 영향을 미칠 수 있습니다.
- 따라서 반드시 **최상위 태그의 className을 컴포넌트 이름과 동일하게 부여한 뒤 그 className하위로 네스팅** 적용해주세요
```suggestion
    <div className="user">
```
다른 부분도 동일하게 적용시켜주세요!

Reply:
넵 수정 완료 했습니다